### ISSUE TITLE
Include a summary of the training set (and test set) in the html file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/wnd-charm#20]
```
Include a summary of the training set (and test set) in the html file, as is 
done on the command line to STDOUT.

Just put what is printed to STDOUT in the html file as well, like:

Processing test set 'worm_part_classifier.fit'.
----------
Summary of 'worm_part_classifier.fit' (2036 samples total, 1 samples per image):
'Class label' number of samples.
'NewBackground' 197
'body'  828
'headALL'   739
'tails' 272
----------
Max balanced training: 197
samples per image=1, training images: 148, testing images 49



```

Original issue reported on code.google.com by `christop...@nih.gov` on 14 Mar 2011 at 5:52
